site/styles/_meta-detail.scss

```diff
@@ -172,6 +175,10 @@
     .live {
       display: none;
     }
+
+    .s72-image {
```

Comment:
Bonus points if you fix this in this PR.  I just noticed [this s72-image](https://github.com/shift72/core-template/blob/b359371cca68a2a1f11f2919898c8272cffb6cb0/site/templates/film/item.jet#L39) ([and this one](https://github.com/shift72/core-template/blob/b359371cca68a2a1f11f2919898c8272cffb6cb0/site/templates/tv/detail.jet#L29)) is missing a closing tag which results in a double wrapped s72-image:
![image](https://user-images.githubusercontent.com/10665859/201546771-66eb6cec-6814-4a74-a645-8fc83b4a1b2f.png)